src/Data/Formatter/Number.purs

```diff
@@ -90,7 +90,9 @@ format ∷ Formatter → Number → String
 format (Formatter f) num =
   let
     absed = Math.abs num
-    tens = if absed > 0.0 then Int.floor $ Math.log absed / Math.ln10 else 0
+    tens = if absed > 0.0
+             then max (Int.floor $ Math.log absed / Math.ln10) 0

```

Comment:
So the issue was that `Math.log` for anything less then 1.0, will return negative number